modules/ROOT/pages/api-gateway-encryption-mule4.adoc

```diff
@@ -0,0 +1,270 @@
+= Gateway Startup Encryption in Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Gateway startup encryption allows you to encrypt sensitive information stored by the runtime such as policies, contracts, and the required credentials to configure the connection between your Mule application and Anypoint Platform.
```

Comment:
According to our style guide, "by the runtime" should be either "by Mule" or "by the Mule runtime engine".
Please check if this change is necessary